torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -252,6 +301,14 @@ void PropagateShapeOnNode(Node * node, bool insert_expands) {
       }
       return;
     }
+    // NB: We assume that all shapes are known within fused kernels
```

Comment:
This seems fragile. The primary place where you might encounter this is in someone trying to optimize the fallback pathway of a failed fusion. But that will be a case where the shapes are not all the same size. It would be better for this to be robust to whatever can be sent here.

Reply:
Actually those cases are no longer needed, so I can just remove them.